armi/runLog.py

```diff
@@ -98,82 +108,79 @@ class Log:
     Abstract class that ARMI code calls to be rendered to some kind of log.
     """
 
-    def __init__(self, verbosity=50):  # pylint: disable=unused-argument
+    def __init__(self, mpiRank=0, mpiSize=1):
         """
         Build a log object
 
         Parameters
         ----------
-        verbosity : int
-            if a msg verbosity is > this, it will be  emitted.
-            The default of 50 means only error messages will be emitted.
-            This usually gets adjusted by user settings quickly after instantiation.
+        mpiRank : int
+            If this is zero, we are in the parent process, otherwise child process.
+            The default of 0 means we assume the parent process.
+            This should not be adjusted after instantiation.
+        mpiSize : int
+            If this is one, we are in the parent process, with child process.
+            This should not be adjusted after instantiation.
```

Comment:
Logger Name.  Sure, I have changed it.

It just seemed natural to have the name be dependent on the child process number because I was trying not to change two current features:

1. Log file names depend on child process (number)
2. Log strings depend on child process (number)

But, as I say, that's an easy change. Done.